Quadriga/src/main/java/edu/asu/spring/quadriga/service/projectblog/IProjectBlogManager.java

```diff
@@ -0,0 +1,39 @@
+package edu.asu.spring.quadriga.service.projectblog;
+
+import java.util.List;
+
+import edu.asu.spring.quadriga.domain.projectblog.IProjectBlog;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.impl.projectblog.ProjectBlogManager;
+
+/**
+ * This interface provides template for project blog manager implementations.
+ * 
+ * @author PawanMahalle
+ *
+ * @see ProjectBlogManager
+ */
+public interface IProjectBlogManager {
+
+    /**
+     * adds new project blog object to database table
+     * <code>tbl_projectblog</code>
```

Comment:
This is the place to mention that the method will also generate an id and set a creation dat.